docs/ensuring-repro/docker/index.md

```diff
@@ -34,6 +34,16 @@ We will also use Docker images to run the analysis modules in the `OpenScPCA-nf`
 
 Follow the instructions to [Install Docker Desktop on Mac](https://docs.docker.com/desktop/install/mac-install/) on the Docker website.
 
+#### Apple silicon (M1-3) Mac users
+
+After installing Docker, we recommend that Apple silicon Mac users follow these additional steps:
+
+1. Install Rosetta 2 [as described in the "System Requirements" section in the "Mac with Apple Silicon" tab](https://docs.docker.com/desktop/install/mac-install/#system-requirements).
+1. Enable Rosetta emulation in the Docker Desktop Settings.
+In the ["General" settings](https://docs.docker.com/desktop/settings/mac/#general) tab, make sure the following are turned **on**:
+    - `Use Virtualization framework`
+    - `Use Rosetta for x86_64/amd64 emulation on Apple Silicon`
```

Comment:
I don't think we need code quotes here, but either way.
```suggestion
    - "Use Virtualization framework"
    - "Use Rosetta for x86_64/amd64 emulation on Apple Silicon"
```

(Could also remove the quotes altogether, but this emphasizes that it isn't our fault that "Silicon" is capitalized here)